mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapper.java

```diff
@@ -170,7 +170,16 @@ public Mono<MailboxCounters> getMailboxCountersReactive(Mailbox mailbox) {
                 }
                 return Mono.just(counters);
             })
-            .doOnNext(counters -> readRepair(mailbox, counters));
+            .doOnNext(counters -> {
+                if (this.cassandraConfiguration.getMailboxReadRepair() > 0) {
```

Comment:
I'm oretty sure it should be

```suggestion
                if (this.cassandraConfiguration.getMailboxCountersReadRepair() > 0) {
```

Reply:
Oh, you mean we should update the name in the original configuration file too? 🤔 
<img width="699" alt="Screen Shot 2023-07-25 at 12 13 15 AM" src="https://github.com/apache/james-project/assets/56495631/5861434b-41cb-4aa3-9d18-d1e610045ea9">

Reply:
I believe what chibenwa is saying is:
- `getMailboxReadRepair()` is used in `CassandraMailboxMapper`
- for `CassandraMessageMapper `, we use `getMailboxCountersReadRepairChanceMax()` and/or `getMailboxCountersReadRepairChanceOneHundred()`

=> for example: https://github.com/apache/james-project/blob/master/mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapper.java#L197

Reply:
Maybe we could extract that line actually in a function to avoid duplication, that we could call from there and from your code change :)